apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -195,17 +195,31 @@ defmodule StateMediator.Integration.GtfsTest do
       assert [%{name: "Braintree - Alewife"}, %{name: "Ashmont - Alewife"}] = shapes_1
     end
 
-    test "Providence/Stoughton has 2 non-ignored shapes each direction" do
+    test "Providence/Stoughton has at least 2 non-ignored shapes in direction 0 and 2 in direction 1" do
       [shapes_0, shapes_1] = shapes_in_both_directions("CR-Providence")
 
       assert [
-               shape_0,
-               %{id: "9890004"},
-               %{id: "SouthStationToStoughtonViaFairmount"}
+               # South Station - Wickford Junction via Back Bay
+               shape_0
+               | remaining_shapes
              ] = shapes_0
 
       assert shape_0.name =~ "South Station - Wickford Junction"
 
+      assert Enum.all?(remaining_shapes, fn shape ->
+               case shape do
+                 # South Station - Stoughton via Back Bay
+                 %{id: "9890004"} -> true
+                 # South Station - Stoughton via Fairmount
+                 %{id: "SouthStationToStoughtonViaFairmount"} -> true
+                 # South Station - Wickford Junction
+                 %{id: "9890009"} -> true
+                 _ -> false
+               end
+             end)
+
+      assert Enum.count(remaining_shapes) >= 1
```

Comment:
Should this assertion be >= 4? (Since the test is for at least 2 outbound shapes and exactly 2 inbound shapes -> at least 4 total shapes)

Reply:
This list is the shapes in direction 0, which have varied in number over time (from 2 <-> 3 <-> 4) for this route. So at least 2 (including the first in the list being the expected one).

Reply:
Oh I see that it's two separate lists now, somehow missed that on the first read-through. Thanks, this makes sense.